src/lib/components/FormModel/children/Form/UI/CheckBox/check-box.component.js

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState, useContext, Fragment } from 'react';
```

Comment:
This is minor, but `Fragment` is no longer used here with this change I think, so it may be removed.

Reply:
Nice catch, I will update this before merging!